cmd/lukso/flags.go

```diff
@@ -398,23 +412,36 @@ func (dependency *ClientDependency) PassStartFlags(ctx *cli.Context) (startFlags
 	name := dependency.name
 	args := ctx.Args()
 	argsLen := args.Len()
+	flagsToSkip := []string{
+		validatorFlag,
+		gethConfigFileFlag,
+		prysmConfigFileFlag,
+		validatorConfigFileFlag,
+	}
+
 	for i := 0; i < argsLen; i++ {
+		skip := false
 		arg := args.Get(i)
-		if arg == fmt.Sprintf("--%s", dependency.name) {
-			continue // when running lukso start --validator we don't want --valdiator falg passed into validator
+		for _, flagToSkip := range flagsToSkip {
+			if arg == fmt.Sprintf("--%s", flagToSkip) {
+				skip = true
+			}
+		}
+		if skip {
+			continue
 		}
 
 		if strings.HasPrefix(arg, fmt.Sprintf("--%s", name)) {
 			if i+1 == argsLen {
-				startFlags = append(startFlags, arg)
+				startFlags = append(startFlags, fmt.Sprintf("--%s", strings.TrimLeft(arg, fmt.Sprintf("--%s-", name))))
 
 				return
 			}
 
 			// we found a flag for our client - now we need to check if it's a value or bool flag
 			nextArg := args.Get(i + 1)
 			if strings.HasPrefix(nextArg, "--") { // we found a next flag, so current one is a bool
-				startFlags = append(startFlags, arg)
+				startFlags = append(startFlags, fmt.Sprintf("--%s", strings.TrimLeft(arg, fmt.Sprintf("--%s-", name))))
```

Comment:
This will produce unwanted results.
Please consult this playground:
https://go.dev/play/p/nk94M5XgY1H